sample-apps/golang/hello.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/gorilla/mux"
+)
+
+func main() {
+
+	port := os.Getenv("APP_PORT")
+	router := mux.NewRouter().StrictSlash(true)
+	router.HandleFunc("/golang", Index)
```

Comment:
Can you get the `golang` value from the `APP_NAME` env var so that this doesn't need to be changed when creating go projects?